homeassistant/components/iotty/manifest.json

```diff
@@ -7,5 +7,5 @@
   "documentation": "https://www.home-assistant.io/integrations/iotty",
   "integration_type": "device",
   "iot_class": "cloud_polling",
-  "requirements": ["iottycloud==0.1.3"]
```

Comment:
Please bump in a separate PR

Reply:
Hi, the new library is necessary for Covers to work, and just making a separate PR that only bumps the lib version seems unnecessary and extra work also for the reviewers

Reply:
No, that's what we want to happen because then we extract what is needed for that library bump to work

Reply:
I'm sorry, I'm not following you on this one. What would change if I made a separate PR before that? Whatever you could do "then" to "extract what is needed for that library bump to work" wouldn't also work in this PR? I'm confused by what you are saying.

Anyway, if you really want me to make a separate PR, fine by me since this is what you want to happen. Just asking to be sure, so it should go like that:

1. I create a new PR that ONLY changes the requirements (so the one line here, and the files in the requirements)
2. Draft this PR
3. Wait for the lib bump PR to be merged
4. Sync with dev
5. Open this PR

Is that correct? Or do you need to do anything else before that?

Reply:
Yep that sounds perfect

Reply:
Here's the new PR: https://github.com/home-assistant/core/pull/125731